Kudu.Core/Jobs/TriggeredJobsManager.cs

```diff
@@ -31,7 +32,18 @@ public TriggeredJobsManager(ITraceFactory traceFactory, IEnvironment environment
 
         public override IEnumerable<TriggeredJob> ListJobs()
         {
-            return ListJobsInternal();
+            var cache = TriggeredJobCache;
+            if (cache == null)
+            {
+                cache = ListJobsInternal();
+                TriggeredJobCache = cache;
+            }
+            return cache;
+        }
```

Comment:
the base declared this method as abstract and it had some implementation before, I am not sure why it was, I will think about it sometime and see if it makes sense to change it.